ozaria/site/views/play/level/LevelPlaybackView.coffee

```diff
@@ -1,6 +1,6 @@
 require('ozaria/site/styles/play/level/level-playback-view.sass')
```

Comment:
Some code has been deleted from this file to get rid of the popup that used to appear on hover of the scrubber in the legacy play level view.